quantum/quantum.c

```diff
@@ -481,3 +481,39 @@ void api_send_unicode(uint32_t unicode) {
 __attribute__((weak)) void startup_user() {}
 
 __attribute__((weak)) void shutdown_user() {}
+
+// Useful string manipulation code
```

Comment:
needs doxygen comments specifically calling out the use of the static conversion buffer.

Reply:
Added.